src/components/PathwayExercise/ExerciseContent/index.js

```diff
@@ -248,26 +232,31 @@ const RenderContent = ({ data, exercise }) => {
             <Typography variant="subtitle1">Code Example</Typography>
           </Box>
           {/* </Toolbar> */}
+
           <Typography
             className={classes.codeWrap}
             dangerouslySetInnerHTML={{
               __html: codeContent,
             }}
           />
-          <Grid container justifyContent="flex-end" mt={2}>
-            <Button
-              variant="contained"
-              color="dark"
-              target="_blank"
-              href={createVisulizeURL(get(data, "value"), data.type, "display")}
-            >
-              Visualize
-            </Button>
-          </Grid>
         </Box>
       </div>
     );
   }
+
+  if (data.component === "code" && data.type === "python") {
+    const pythonCode = data.value
+      .replace(/<br>/g, "\n")
+      .replace(/&emsp;/g, " ")
```

Comment:
Consider adding an object to the constants file that has all of these replacements (See a related comment of why here: https://github.com/navgurukul/sansaar/pull/469#discussion_r771807014). Here, `&emsp;` would be replaced with a number of spaces equal to the value of the `INDENT` constant mentioned earlier.